keel-core/src/test/kotlin/com/netflix/spinnaker/keel/services/ComparableLinksTests.kt

```diff
@@ -245,87 +244,88 @@ class ComparableLinksTests : JUnit5Minutests {
           emptyList()
         }
 
-        every { repository.artifactVersions(releaseArtifact) } returns versions.toArtifactVersions(releaseArtifact)
+        every {
+          repository.getVersionInfoInEnvironment(any(), "test", any())
+        } returns listOf(
+          StatusInfoForArtifactInEnvironment(
+            version0,
+            PREVIOUS,
+            version1,
+            clock.instant().minus(Duration.ofHours(4))
+          ),
+          StatusInfoForArtifactInEnvironment(
+            version1,
+            PREVIOUS,
+            version2,
+            clock.instant().minus(Duration.ofHours(3))
+          ),
+          StatusInfoForArtifactInEnvironment(version2, CURRENT, null, clock.instant().minus(Duration.ofHours(2))),
+        )
+
+        every {
+          repository.getVersionInfoInEnvironment(any(), "staging", any())
+        } returns listOf(
+          StatusInfoForArtifactInEnvironment(
+            version0,
+            PREVIOUS,
+            version1,
+            clock.instant().minus(Duration.ofHours(4))
+          ),
+          StatusInfoForArtifactInEnvironment(version1, CURRENT, null, clock.instant().minus(Duration.ofHours(2))),
+        )
       }
 
       context("stash") {
         before {
```

Comment:
Removed a lot of mocks like this because we no longer call the db for this info, we pull from the list of artifact version info that we load once.

Reply:
woohoo!